subpar.bzl

```diff
@@ -172,7 +172,7 @@ parfile_test = rule(
 You probably want to use par_test() instead of this.
 """
 
-def par_binary(name, **kwargs):
+def par_binary(name, compiler="//compiler:compiler.par", **kwargs):
```

Comment:
You should be able to get rid of the `compiler=...` here, in favor of adding `compiler = kwargs.get('compiler', '//compiler:compiler.par')` around line 196.

Also, I'd like to figure out how to avoid repeating that label twice, once here and once in `parfile_attrs` above.

Reply:
Done